cl/recap/management/commands/nightly_pacer_updates.py

```diff
@@ -242,4 +265,30 @@ def handle(self, *args, **options):
                 args=(d.pk, d.court_id), queue=queue
             )
 
+        for i, rd_id in enumerate(get_rd_ids_pray_and_pray()):
+
+            if i % 10 == 0:
+                logger.info(
+                    "Sent %s RECAP Documents to celery for crawling so far.", i
+                )
+
+            rd = (
+                RECAPDocument.objects.select_related("docket_entry__docket")
+                .only(
+                    "pacer_doc_id",
+                    "docket_entry_id",
+                    "docket_entry__docket_id",
+                    "docket_entry__docket__court_id",
+                    "is_sealed",
+                )
```

Comment:
The code after this query shows we only need the `pacer_doc_id` and the recap document's id.

Reply:
> These are the fields used in `check_prayer_pacer` as well. 

`check_prayer_pacer` is a Celery task, so it runs in a separate context. Instead of passing the entire object, we only pass the record ID. This allows the task to fetch the specific recap document it needs when it's executed by a Celery worker.

Reply:
> Do you think it is inefficient to directly pass the record into check_prayer_pacer?

Yes! Passing just the ID is definitely the way to go. When we schedule tasks, we store the necessary context in Redis so our Celery workers can grab it later. If we start pushing full records, we'll fill up Redis memory in no time. Plus, think about what happens if we update a record after scheduling the task. The Celery worker would be working with outdated information. Fetching the record within the Celery task ensures we're always using the most current data.